File: programacao/python/respostas/selecao/15.py
p1 = float(input("Digite a nota da p1: "))
p2 = float(input("Digite a nota da p2: "))
p3 = float(input("Digite a nota da p3: "))

media = (p1 + p2 + p3) / 3.0

if media > 6 and p1 > 0 and p2 > 0 and p3 > 0:
    print("Aprovado para a segunda fase")
else:
    print("Nao aprovado")

File: programacao/c/respostas/selecao/15.c
#include <stdio.h>

int main(void) {
    float p1, p2, p3, media;

    printf("Digite as tres notas: ");
    scanf("%f %f %f", &p1, &p2, &p3);

    media = (p1 + p2 + p3) / 3.0;

    if (media > 6 && p1 > 0 && p2 > 0 && p3 > 0) {
        printf("Aprovado para a segunda fase\n");
    } else {
        printf("Nao aprovado\n");
    }

    return 0;
}